tests/test_views.py

```diff
@@ -15,6 +15,21 @@ def test_profile_detail(client, db, profile, user):
     assert result["username"] == user.username
 
 
+def test_profile_detail_trimmed(client, db, profile, user):
+    response = client.get(f"/profiles/trimmed/{profile.pk}/")
+    result = response.json()
+    assert response.status_code == 200, result
+    assert "username" not in result
```

Comment:
Not a blocking change, but I'd argue that since this is testing the `only` keyword, we should be checking to ensure that the only key in the response is the `id` key, not just that a particular other key isn't present.